relayer-cli/src/components.rs

```diff
@@ -71,27 +75,29 @@ pub fn enable_ansi() -> bool {
 /// Builds a tracing filter based on the input `log_level`.
 /// Enables tracing exclusively for the relayer crates.
 /// Returns error if the filter failed to build.
-fn build_tracing_filter(log_level: String) -> Result<EnvFilter, FrameworkError> {
-    let target_crates = ["ibc_relayer", "ibc_relayer_cli"];
-
-    // SAFETY: unwrap() below works as long as `target_crates` is not empty.
-    let directive_raw = target_crates
-        .iter()
-        .map(|&c| format!("{}={}", c, log_level))
-        .reduce(|a, b| format!("{},{}", a, b))
-        .unwrap();
+fn build_tracing_filter(log_level: LogLevel) -> Result<EnvFilter, FrameworkError> {
+    let directive = std::env::var(HERMES_LOG_VAR).unwrap_or_else(|_| {
+        let target_crates = ["ibc_relayer", "ibc_relayer_cli"];
```

Comment:
I see here we're using `RUST_LOG` not only to replace the log level, but also the crates/modules. Can we give an example on how we're supposed to use `RUST_LOG` to overwrite the tracing parameters?

Reply:
Yes, using `RUST_LOG` will override everything. I'll add an example, perhaps something like:

```RUST_LOG=tendermint_rpc=debug,ibc_relayer=info,ibc_relayer_cli=info hermes start```